lib/model/internal_link.dart

```diff
@@ -195,9 +193,22 @@ Narrow? _interpretNarrowSegments(List<String> segments, PerAccountStore store) {
     }
   }
 
-  if (isMentionedElement != null) {
+  if (isElementOperands.isNotEmpty) {
     if (streamElement != null || topicElement != null || dmElement != null) return null;
-    return const MentionsNarrow();
+    switch (isElementOperands.singleOrNull) {
```

Comment:
Let's make this more explicit:
```suggestion
    if (isElementOperands.length > 1) return null;
    switch (isElementOperands.single) {
```

I actually started writing a comment asking if the null case was impossible, because this is inside an `isNotEmpty` conditional. Then I realized there are two different cases that `singleOrNull` turns to null.